src/north/ApiHandler.class.js

```diff
@@ -55,6 +59,26 @@ class ApiHandler {
   connect() {
     const { name, api } = this.application
     this.logger.info(`North API ${name} started with protocol ${api}`)
+    this.statusData['Connected at'] = new Date().toISOString()
+    this.updateStatusDataStream()
```

Comment:
I disagree with this : some connectors do not 'connect' like fileWriter for example. Moreover, when a connector connects to something, you should not update this status here : the connection occurs generally on the driver (like for protocols) so you should update the statusData in the callback function instead for each api (when applicable)